2022-04-13 02:44:07,435 INFO    MainThread:2221 [wandb_setup.py:_flush():69] setting env: {}
2022-04-13 02:44:07,435 INFO    MainThread:2221 [wandb_setup.py:_flush():69] setting login settings: {}
2022-04-13 02:44:07,436 INFO    MainThread:2221 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220413_024407-gkn7xjf3/logs/debug.log
2022-04-13 02:44:07,436 INFO    MainThread:2221 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220413_024407-gkn7xjf3/logs/debug-internal.log
2022-04-13 02:44:07,436 INFO    MainThread:2221 [wandb_init.py:init():369] calling init triggers
2022-04-13 02:44:07,437 INFO    MainThread:2221 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-04-13 02:44:07,437 INFO    MainThread:2221 [wandb_init.py:init():418] starting backend
2022-04-13 02:44:07,437 INFO    MainThread:2221 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-13 02:44:07,477 INFO    MainThread:2221 [backend.py:ensure_launched():123] starting backend process...
2022-04-13 02:44:07,500 INFO    MainThread:2221 [backend.py:ensure_launched():128] started backend process with pid: 2247
2022-04-13 02:44:07,505 INFO    MainThread:2221 [wandb_init.py:init():423] backend started and connected
2022-04-13 02:44:07,512 INFO    MainThread:2221 [wandb_init.py:init():465] updated telemetry
2022-04-13 02:44:07,516 INFO    MainThread:2221 [wandb_init.py:init():484] communicating current version
2022-04-13 02:44:10,126 INFO    MainThread:2221 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-13 02:44:10,127 INFO    MainThread:2221 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-04-13 02:44:11,740 INFO    MainThread:2221 [wandb_init.py:init():522] starting run threads in backend
2022-04-13 02:44:16,750 INFO    MainThread:2221 [wandb_run.py:_console_start():1524] atexit reg
2022-04-13 02:44:16,752 INFO    MainThread:2221 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-04-13 02:44:16,765 INFO    MainThread:2221 [wandb_run.py:_redirect():1376] Redirecting console.
2022-04-13 02:44:16,771 INFO    MainThread:2221 [wandb_run.py:_redirect():1438] Redirects installed.
2022-04-13 02:44:16,771 INFO    MainThread:2221 [wandb_init.py:init():546] run started, returning control to user process
2022-04-13 02:44:18,287 INFO    MainThread:2221 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-04-13 02:44:18,290 INFO    MainThread:2221 [wandb_run.py:_restore():1466] restore
2022-04-13 02:44:21,206 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 1046
}

2022-04-13 02:44:21,328 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 1046
}

2022-04-13 02:44:21,787 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 1046
}

2022-04-13 02:44:22,400 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 7132
}

2022-04-13 02:44:22,506 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6818
  total_bytes: 7132
}

2022-04-13 02:44:22,611 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7132
  total_bytes: 7132
}

2022-04-13 02:44:22,717 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7132
  total_bytes: 7132
}

2022-04-13 02:44:22,824 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7132
  total_bytes: 7132
}

2022-04-13 02:44:22,930 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7132
  total_bytes: 7132
}

2022-04-13 02:44:23,035 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7132
  total_bytes: 7132
}

2022-04-13 02:44:23,140 INFO    MainThread:2221 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7132
  total_bytes: 7132
}

2022-04-13 02:44:24,937 INFO    MainThread:2221 [wandb_run.py:_show_files():1838] logging synced files
